SecurityService.OpenIdConnect.IntegrationTests/Common/Hooks.cs

```diff
@@ -58,18 +64,41 @@ public void AfterScenario()
         [BeforeScenario(Order = 0)]
         public async Task BeforeScenario()
         {
-            ChromeOptions options = new ChromeOptions();
-            options.AddArguments("--disable-gpu");
-            options.AddArguments("--no-sandbox");
-            options.AddArguments("--disable-dev-shm-usage");
-            var experimentalFlags = new List<String>();
-            experimentalFlags.Add("same-site-by-default-cookies@2");
-            experimentalFlags.Add("cookies-without-same-site-must-be-secure@2");
-            options.AddLocalStatePreference("browser.enabled_labs_experiments", experimentalFlags);
-            this.WebDriver = new ChromeDriver(options);
+            String? browser = Environment.GetEnvironmentVariable("Browser");
+            //browser = "Edge";
+
+            if (browser == null || browser == "Chrome")
+            {
+                ChromeOptions options = new ChromeOptions();
+                options.AddArguments("--disable-gpu");
+                options.AddArguments("--no-sandbox");
+                options.AddArguments("--disable-dev-shm-usage");
+                var experimentalFlags = new List<String>();
+                experimentalFlags.Add("same-site-by-default-cookies@2");
+                experimentalFlags.Add("cookies-without-same-site-must-be-secure@2");
+                options.AddLocalStatePreference("browser.enabled_labs_experiments", experimentalFlags);
+                this.WebDriver = new ChromeDriver(options);
+            }
+
+            if (browser == "Firefox")
+            {
+                FirefoxOptions options = new FirefoxOptions();
+                options.AddArguments("-headless");
+                this.WebDriver = new FirefoxDriver(options);
+            }
+
+            if (browser == "Edge")
+            {
+                EdgeOptions options = new EdgeOptions();
+                //EdgeDriverService service = EdgeDriverService.CreateDefaultService(@"D:\Program Files (x86)\EdgeDriver\", "msedgedriver.exe");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6248858)